README.md

```diff
@@ -1,32 +1,148 @@
+[23 phases](https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html#Lifecycle_Reference)
+
+The image below shows the first 15 phases up to **test**.
+The phases are executed sequentially.  If the `test` phase is
+executed, all phases before it must execute.
+
+![default life cycle phases](https://curriculum-content.s3.amazonaws.com/6002/creating-a-maven-project/defaultlifecycle.png)
+
+Consider what must happen to run a Junit test.
+The Junit test class must be compiled before we can
+execute the test. The compiled code goes in a
+test directory that must be generated prior to compilation.
+The compiler also performs some post-compilation processing
+to optimize the bytecode.
+Since a Junit test class is designed to test the
+methods of a source code class, the source
+code must also be compiled, necessary resources
+generated, and post-compilation processing performed. 
+
+It would be tedious to do all of these steps
+manually.  A build system automates
+much of the process by providing a
+lifecycle of clearly defined phases.
+
+Let's explore some of this using IntelliJ's Maven Tool Window,
+which can be opened either by clicking on "Maven" along
+the edge of the IntelliJ window, or by selecting ** View | Tool Windows | Maven**
+from the main menu.
+  
+![maven tool window](https://curriculum-content.s3.amazonaws.com/6002/creating-a-maven-project/maventoolwindow.png)
+
+Expand the Lifecycle folder to see the phases available for execution. 
+The *clean* and *site* phases are not part of the default lifecycle, but
+rather phases from other built-in lifecycles.
+
+### Plugins and Goals
+
+Expand the Plugin folder to see the plugins and goals.
+When we run a phase, all goals bound to the phase are executed.
+
+For example:
+- **compiler:compile** - *compile* goal from
+  the *compiler* plugin is bound to the *compile* phase
+- **compiler:testCompile** - *testCompile* goal from
+  the *compiler* plugin is bound to the *test-compile* phase
+
+Each plugin has a help goal.  Select the **clean:help** goal and then
+press the green run button at the top of the Maven build window
+to execute the goal.  The console will display documentation
+about the two goals **clean:clean** and **clean:help** that are
+part of the Apache Maven clean plugin.
+
+![clean plugin help](https://curriculum-content.s3.amazonaws.com/6002/creating-a-maven-project/cleanhelp.png)
+
+A phase may consist of multiple goals. Running a single goal
+does not run the entire phase, nor does it run the goals
+in the preceding phases. 
+  
+To execute all goals in a phase (and preceding phases),
+select the phase in the LifeCycle folder and press run.
+
+![running a phase](https://curriculum-content.s3.amazonaws.com/6002/creating-a-maven-project/runningphase.png)
+ 
+
+## Building a project
+
+Let's see how IntelliJ builds the project when we change the code
+and need to recompile. 
+
+Edit the `main` method to print "Hello" instead of "Hello World!", then:
```

Comment:
Nit: print "Hello!" instead of just "Hello" since it looks like in the screenshot below it has the exclamation point still attached. Probably just easier to change it here than having to re-do the screenshot.

Reply:
good catch, thanks.